runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -1300,6 +1300,24 @@ private Image screenshot()
 			height = dim.height;
 		}
 
+		String gdkScaleEnv = System.getenv("GDK_SCALE");
+		if (gdkScaleEnv != null)
+		{
+			try
+			{
+				int gdkScale = Integer.parseInt(gdkScaleEnv);
+				if (gdkScale > 1)
+				{
+					width = width * gdkScale;
+					height = height * gdkScale;
+				}
+			}
+			catch (NumberFormatException e)
+			{
+				log.info("GDK_SCALE is not valid");
```

Comment:
this should probably be warn or debug. Also have you tested this with stretched mode?

Reply:
Yeah it works fine with stretched mode
![2020-05-23_20-27-16](https://user-images.githubusercontent.com/33142989/82738977-e9d27c00-9d33-11ea-9227-546439cb3f69.png)
![2020-05-23_20-27-18](https://user-images.githubusercontent.com/33142989/82738976-e63ef500-9d33-11ea-9903-3bf928ea6dba.png)
![2020-05-23_20-27-16](https://user-images.githubusercontent.com/33142989/82739002-16869380-9d34-11ea-9aa2-6e2206dda92a.png)